runtime/dolphin/src/lib.rs

```diff
@@ -869,6 +869,12 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_manta_pay::runtime::PullRuntimeApi<Block> for Runtime {
+		fn pull(checkpoint: pallet_manta_pay::RawCheckpoint) -> pallet_manta_pay::PullResponse {
```

Comment:
at RT api level too

Reply:
I think `pull` here is proper. Since the RPC entrance is `mantaPay_pull`. The semantics is quite clear.

Reply:
if you look at the other APIs consensus seems to be to name the API like its callee e.g. `collect_collation_info`
But as you want, I'm ok without it

Reply:
Open for better naming suggestion for sure! This is a change will cause front-end/wallet/signer change as well. I feel like we need to do a proper API doc.

Reply:
i would just follow precedent and name the API `pull_ledger_diff`.
If that is too big a change i'm fine with reverting back to the old `pull` naming for now

Reply:
I agree `pull_ledger_diff` is a better name, let's change it across the board.